screens/AddUserToChatScreen.js

```diff
@@ -0,0 +1,220 @@
+import React from 'react';
+import { connect } from 'react-redux';
+import { bindActionCreators } from 'redux';
+import {
+    Platform,
+    ScrollView,
+    StyleSheet,
+    TouchableOpacity,
+    View,
+    KeyboardAvoidingView,
+    AsyncStorage
+} from 'react-native';
+import {
+    Container,
+    Header,
+    Title,
+    Content,
+    Form,
+    Label,
+    Button,
+    Left,
+    Right,
+    Body,
+    Icon,
+    Text,
+    List,
+    ListItem,
+    Item,
+    Input,
+    Toast
+} from 'native-base';
+
+import { baseUrl } from '../constants/api';
+
+import { putUserInChat } from '../actions/chats';
+import {
+    handleResponse,
+    addAuthHeader
+} from '../helpers/api';
+
+const AddUserForm = ({ addUserToChat, cid: chatId, user }) => {
+    const { 
+        id: userId, 
+        display_name: displayName
+    } = user;
+    return (
+        <Form style={styles.contentContainer}>
+            <Item stackedLabel>
+                <Label>User display name</Label>
+                <Text>
+                    { displayName }
+                </Text>
+            </Item>
+            <Button 
+                full
+                onPress={() => addUserToChat(chatId, userId)}>
+                <Text>Add To Chat</Text>
+            </Button>
+        </Form>
+    );
+}
+
+class AddUserToChatScreen extends React.Component {
+    static navigationOptions = {
+        header: null
+    };
+
+    state = {
+        email: '',
+        foundUser: null,
+        error: ''
+    };
+
+    _submitForm = async () => {
+        const { email } = this.state;
```

Comment:
i think you'll want to reset error to '' using setState because if the user doesn't exist, an error will show and then still show when you search for another email